addons/l10n_it_edi/models/account_move.py

```diff
@@ -450,6 +450,18 @@ def _l10n_it_edi_grouping_function_total(self, base_line, tax_data):
         skip = tax_data['is_reverse_charge'] or self._l10n_it_edi_is_neg_split_payment(tax_data)
         return not skip
 
+    def _prepare_product_base_line_for_taxes_computation(self, product_line):
```

Comment:
Add a docstring with a simple explanation like `Roundings must appear as lines in the XML` or whatever.